### ISSUE TITLE
Attention % rate on Twitter publisher gets increases only after browser restart

### ISSUE THREAD

**Issue by GeetaSarvadnya** [brave/brave-browser#23665]
## Description 

Found while testing https://github.com/brave/brave-browser/issues/23332
Attention % rate on Twitter publisher gets increases only after browser restart
## Steps to Reproduce

   1. Clean profile 1.41.77
   2. Enable rewards
   3. Open a Twitter pubs  https://twitter.com/bravelaurenwags
   4. Stay on the Twitter pubs page for 20 secs
   5. Open the BR panel and see attention % is still 0
   6. Restart the browser via the hamburger menu
   7. Repeat the step 4
   8. Attention % rate on Twitter pubs gets increased 

## Actual result:

Attention % rate on Twitter publisher gets increases only after browser restart

## Expected result:
Attention % rate on Twitter publisher should get increased when user stays on the page for 10 secs

## Reproduces how often: 

Easy
## Brave version (brave://version info)

Brave | 1.41.77 Chromium: 103.0.5060.53 (Official Build) beta (64-bit)
-- | --
Revision | a1711811edd74ff1cf2150f36ffa3b0dae40b17f-refs/branch-heads/5060@{#853}
OS | Windows 10 Version 21H2 (Build 19044.1766)

## Version/Channel Information:

- Can you reproduce this issue with the current release? Yes
- Can you reproduce this issue with the beta channel? Yes
- Can you reproduce this issue with the nightly channel? Not sure

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? NA
- Does the issue resolve itself when disabling Brave Rewards? NA
- Is the issue reproducible on the latest version of Chrome? NA

## Miscellaneous Information:

cc: @brave/qa-team @emerick @Miyayes 

Looks like Windows 10 specific issue, @srirambv tried on Windows 11 and it works for him

**Comment by LaurenWags** [comment]
Duplicate of https://github.com/brave/brave-browser/issues/13391
